test.galaxyproject.org/machine_learning.yml.lock

```diff
@@ -0,0 +1,155 @@
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+tool_panel_section_label: Machine Learning
+tools:
+- name: model_prediction
+  owner: bgruening
+  revisions:
+  - 6efb9bc6bf32
+  tool_panel_section_id: statistics
+  tool_panel_section_label: Statistics
```

Comment:
```suggestion
  tool_panel_section_id: machine_learning
  tool_panel_section_label: Machine Learning
```
Can you fix this and the other entries in this lock file? Thanks!

Reply:
Fixed. I call this the "Curse of Copy-Paste": Any time saved by copy/paste'ing will be lost by trouble shooting the errors introduced by copy-paste'ing :)